whoisit/bootstrap.py

```diff
@@ -100,14 +104,15 @@ def clear_bootstrapping(self):
         self._use_iana_overrides = False
         log.debug('Cleared bootstrap data')
 
-    def bootstrap(self, overrides=False, allow_insecure=False):
+    def _bootstrap(self, overrides=False, allow_insecure=False):
         if self.is_bootstrapped():
             return True
         self._use_iana_overrides = bool(overrides)
         self._allow_insecure = bool(allow_insecure)
         items_loaded = set()
         for name, url in self.BOOTSTRAP_URLS.items():
-            response = http_request(self.session, url)
+            response = yield url
+            yield
```

Comment:
Is this extra blank yield needed?

Reply:
Yes, that is the generator magic I mentioned. It's to handle the send control flow in the generator loop in the subclass bootstrap command. I did similar things in the __init__ function

Reply:
OK, makes sense, thanks!